schema.sql

```diff
@@ -0,0 +1,88 @@
+DROP TABLE IF EXISTS application;
+DROP TYPE IF EXISTS application_status;
+DROP TABLE IF EXISTS applicant_phone_number;
+DROP TYPE IF EXISTS phone_type;
+DROP TABLE IF EXISTS applicant_address;
+DROP TABLE IF EXISTS applicant;
+DROP TABLE IF EXISTS advisor;
+DROP TYPE IF EXISTS role;
+DROP TABLE IF EXISTS "user";
+
+CREATE TABLE "user"
+(
+    id       BIGSERIAL,
+    email    TEXT NOT NULL,
+    username TEXT NOT NULL,
+    CONSTRAINT user_PK PRIMARY KEY (id),
+    CONSTRAINT user_email_UQ UNIQUE (email),
+    CONSTRAINT user_email_CH CHECK (email LIKE '%@%'),
+    CONSTRAINT user_username_UQ UNIQUE (username)
+);
+
+CREATE TYPE role AS ENUM ('associate', 'partner', 'senior');
+
+CREATE TABLE advisor
```

Comment:
as I know the best practice table name use plural

Reply:
There no any conventions